index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="style.css" rel="stylesheet">
+  <title>Product Landing Page</title>
+</head>
+<body>
+  <header id="header" class="container">
+    <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" alt="Ori" id="header-img">
+    <nav id="nav-bar">
+      <ul>
+          <li>
+            <a class="nav-link" href="#features">Features</a>
+          </li>
+          <li>
+            <a class="nav-link" href="#how-it-works">How It Works</a>
+          </li>
+          <li>
+            <a class="nav-link" href="#pricing">Pricing</a>
+          </li>
+      </ul>
+    </nav>
+  </header>
+  <!-- <iframe id="video" width="560" height="315" src="https://www.youtube.com/embed/y8Yv4pnO7qc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe> -->
+  
+ <!--  <section id="newsletter">
+    <form action="https://www.freecodecamp.com/email-submit" id="form">
+      <input name="email" type="email" id="email" placeholder="Enter Email Address">
+      <input type="submit" id="submit">
+    </form>
+  </section> -->
```

Comment:
Try to avoid Block Comments of code in your work. If you won't need them you can just delete them.